Source/ActivityPub.Types/AS/Collection/ASOrderedCollection.cs

```diff
@@ -113,8 +113,14 @@ public int TotalItems
     /// <summary>
     ///     True if this is a paged collection, false otherwise.
     /// </summary>
+    [MemberNotNullWhen(true, nameof(FirstPopulated))]
     public bool IsPaged => Current != null || First != null || Last != null;
 
+    /// <summary>
+    ///     In a paged Collection, gets the first populated entity.
+    /// </summary>
+    public Linkable<ASOrderedCollectionPage>? FirstPopulated => IsPaged ? Current ?? First ?? Last : null;
```

Comment:
same thing

Reply:
I guess I just had the IsPaged check there for readability, but totally makes sense to remove it. Just pushed up these fixes.